tests/puppeteer/pages/BO/customers.js

```diff
@@ -1,6 +1,6 @@
 const CommonPage = require('../commonPage');
 
-module.exports = class BO_CUSTOMERS extends CommonPage {
+module.exports = class CUSTOMERS extends CommonPage {
```

Comment:
```suggestion
module.exports = class Customer extends CommonPage {
```
We shouldn't put names in plural I think.
Also, following PSR-1 standards, we must write class names in `StudlyCaps` (https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md#3-namespace-and-class-names)